Removing a Sync Connection (#354)
owncloud/docs-client-desktop: modules/ROOT/pages/navigating.adoc

* Files are removed that cause individual errors three times during a synchronization. However, the client provides the option of retrying a synchronization three additional times on files that produce errors.

For more detailed information see the xref:architecture.adoc#ignored-files[Ignored Files section].

== Removing a Sync Connection

> When removing a synchronisation connection, the connection will get removed but all data synchronized will stay locally and does not get deleted. To remove a sync relationship, change to the account and select the synchornisation relationship that should get removed. When clicking the btn:[...] (three dots) icon, the following window appears:

Reviewer: ```suggestion
When removing a synchronization connection, all synchronized data will stay locally and not get deleted. To remove a sync relationship, change to the account and select the synchronization relationship that should get removed. When clicking the btn:[...] (three dots) icon, the following window appears:
```

image::navigating/remove_sync_connection.png[Removing a Sync Relationship,width=250,pdfwidth=60%]

Select btn:[Remove folder sync connection] to remove it. If you also want to delete the related data, go to the folder and delete it.

TIP: Before removing the connection, click btn:[Show in Explorer]. This will open the source folder where all the synced data is located. This eases deleting this folder as you do not need to search for it after removing the sync connection.

> On Windows, you might be asked for elevated permissions to delete that folder. Follow the procedure shown or ask you administrator for assistance.

Reviewer 2: Still normally it should not ask.

Reviewer 2: Can't reproduce:
https://github.com/owncloud/client/issues/10439

> On Windows, you might be asked for elevated permissions to delete that folder. Follow the procedure shown or ask you administrator for assistance. In rare cases, the folder denies deletion even if elevated permissions were given. In such a case, mark that folder in Explorer and press btn:[Shift + Delete] together and follow the procedure shown. This will delete the folder without using the trash bin.

Reviewer 2: Are you able to reproduce this behavior? This wasn't clear from your other ticket.
If that is really the case it is indeed something we might be able to improve on....

Author: This was the only possibility to delete the folder. I was wondering why the folder was not deletable and tried many things. This was the only one that worked, though there may be in depth ways in PowerShell I have not tried so far. I was not able to reproduce even doing absolutely the same thing many times. But one time is too much and such a tip can help users - and it is a standard Windows command...

Reviewer 3: > But one time is too much

One-time things that can't be reproduced and aren't understood are nothing for the docs. Sufficient to discuss in Central post or GH issue.

Author: ok, will remove

Author: @michaelstingl @TheOneRing I have updated the text, pls re-check.

> On Windows, you may get asked for elevated permissions deleting that folder. Follow the procedure shown or ask you administrator for assistance. In rare cases, the folder denies deletion even elevated permissions were given. In such a case, mark that folder in Explorer and press btn:[Shift + Delete] together and follow the procedure shown. This will delete that folder without using the trashbin.

Reviewer: ```suggestion
On Windows, you might be asked for elevated permissions to delete that folder. Follow the procedure shown or ask you administrator for assistance. In rare cases, the folder denies deletion even if elevated permissions were given. In such a case, mark that folder in Explorer and press btn:[Shift + Delete] together and follow the procedure shown. This will delete the folder without using the trash bin.
```
